configurate-core/src/main/java/ninja/leaping/configurate/objectmapping/ObjectMapper.java

```diff
@@ -75,9 +75,9 @@
      * Holder for field-specific information
      */
     protected static class FieldData {
-        private final Field field;
-        private final TypeToken<?> fieldType;
-        private final String comment;
+        final Field field;
+        final TypeToken<?> fieldType;
+        protected final String comment;
```

Comment:
I'd rather keep these private and use encapsulation with public getters. That allows projects that override/wrap the object mapper field data to make use of these fields rather than capture them on their own, and doesn't expose the fields themselves.